libzecale/circuits/fields/fp6_3over2_gadgets.tcc

```diff
@@ -83,6 +112,37 @@ Fp6_3over2_variable<Fp6T> Fp6_3over2_variable<Fp6T>::operator+(
             other.annotation_prefix.c_str()));
 }
 
+template<typename Fp6T>
+Fp6_3over2_variable<Fp6T> Fp6_3over2_variable<Fp6T>::operator-(
+    const Fp6_3over2_variable<Fp6T> &other) const
+{
+    return Fp6_3over2_variable<Fp6T>(
+        this->pb,
+        _c0 - other._c0,
+        _c1 - other._c1,
+        _c2 - other._c2,
+        FMT(this->annotation_prefix, " - a"));
```

Comment:
What is `a` in the annotation? Shouldn't it be `-other`?
Moreover, why did you add a space between `-` and `a`? the space is the annotation separator, right?

Reply:
changed